Pylint: Fix `consider-merging-isinstance` issue

## 💡 Description
Update the codebase to use a single call to `isinstance` instead of multiple ones. `pylint` report is attached (`pylint . --disable=all --enable=consider-merging-isinstance`):
```
************* Module pds.naif_pds4_bundler.classes.list
src/pds/naif_pds4_bundler/classes/list.py:260:43: R1701: Consider merging these isinstance calls to isinstance(value, (dict, list)) (consider-merging-isinstance)
```

--- a/src/pds/naif_pds4_bundler/classes/list.py
+++ b/src/pds/naif_pds4_bundler/classes/list.py
@@ -257,9 +257,7 @@ class KernelList:
                                                     f"Consider reviewing your NPB setup."
                                                 )
 
-                                        if isinstance(value, list) or isinstance(
-                                            value, dict
-                                        ):
+                                        if isinstance(value, (list, dict)):
                                             handle_npb_error(
                                                 f"-- Kernel {kernel} description could not be updated with "
                                                 f"pattern."

